aiidalab_widgets_base/viewers.py

```diff
@@ -305,7 +326,6 @@ def change_supercell(_=None):
 
         return ipw.VBox([supercell_selector, background_color, center_button])
 
-    @observe("cell")
```

Comment:
Instead of using the decorator, observers are registered manually only when the corresponding tab is displayed.

Reply:
I don't think it is a good idea. `cell` is the `_StructureDataBaseViewer` trait. And, as I wrote in another comment, all the sub-widgets should always exist (despite they are not always shown).

Reply:
Okay, Done.